res/graphics/police type graphics/carabinieri.svg

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
If you have Inkscape installed, can you open these SVG files and save them as "Optimized SVG" (with the suggested default settings)? That removes some unneeded / Inkscape-specific stuff from the files.

Reply:
You are my wiki :laughing: Thanks, I didn't know Inkscape had that, usually I just remove tags manually :laughing: Will optimize also the quest icon then 😁